Makefile.am

```diff
@@ -721,6 +732,82 @@ check-recursive: eslint
 check: check-for-system-nss prettier eslint check-recursive
 	$(GEN_COVERAGE_COMMAND)
 
+perftest_workdir = $(abs_builddir)/perftest/workdir
+perftest_pid_file = $(perftest_workdir)/coolwsd.pid
+perftest_coolwsd_log = $(perftest_workdir)/coolwsd.log
+perftest_coolwsd_output = $(perftest_workdir)/coolwsd_output.log
+perftest_ldflags = -module -rpath /dummy
+all_perftests_la =
+
+all_perftests_la += perftest/sample/SamplePerfTest.la
+perftest_sample_SamplePerfTest_la_SOURCES=perftest/sample/SamplePerfTest.cpp
+perftest_sample_SamplePerfTest_la_LDFLAGS=$(perftest_ldflags)
+all_perftests_la += perftest/example/ExamplePerfTest.la
```

Comment:
Hmm, now we have both sample and example perf tests. :-) I would suggest to came up with a real workload, just a few, 2-4 ones and use them to demonstrate that the framework is working. As mentioned earlier, a minimal feature set is fine, but you usually should not add code that you'll remove in the next commit.

In general, please try to address the unchecked points from https://github.com/CollaboraOnline/online/pull/9025#issuecomment-2104742251, I would like to approve this rather sooner than later, but not at the cost of just forgetting about all review feedback.

As mentioned offline yesterday, I can live with the current "call out to bash" code for now, if they are clearly annotated with some `// TODO(neilguertin) don't call out to bash` markup. Thanks!